aws-lambda-router/cmd/main.go

```diff
@@ -63,6 +66,9 @@ func main() {
 			}),
 		),
 	)
+	if err != nil {
```

Comment:
why is this extra check needed?

Reply:
hmm, before this change it was loading the config and it didn't check for errors at this stage

Reply:
James is correct, but not everything will return an error.